plugins/analysis/dashboard/frontend/package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "goshimmer-analysis-dashboard",
+  "name": "goshimmer-analysisDashboard",
```

Comment:
why not camelCase everything?

Reply:
I think we should rather not touch the frontend code at all. Changed